index.js

```diff
@@ -3,30 +3,58 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let i = {};
+  while (i< rolls.length){
+    if(rolls[i] === toequal.rolls)
+    return false
+    i++
+  }
+  return true
+}
+
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  let result = 0
+  for(let value of rolls){
+    result += value;
```

Comment:
read the instructions carefully. You are give na `value` and you are looking for that number in the rolls array. Therefore, there is no addition to be done. 

Instead you'd have a conditional in your loop that checks if the element is equal to the value that you are looking for.